.github/workflows/doc.yml

```
@@ -53,6 +53,27 @@ jobs:
     - name: 'Build HTML documentation'
       run: make -C Doc/ SPHINXOPTS="-q" SPHINXERRORHANDLING="-W --keep-going" html
 
+    # Add pull request annotations for Sphinx nitpicks (missing references)
+    - name: Get list of changed files
+      id: changed_files
+      uses: Ana06/get-changed-files@v2.2.0
+    - name: 'Build changed files with nitpicks'
+      continue-on-error: true
+      run: |
+        # Mark files the pull request modified
+        touch ${{ steps.changed_files.outputs.added_modified }}
+        # Build docs with the '-n' (nit-picky) option, convert warnings
```

```suggestion
        # Build docs with the '-n' (nit-picky) option; convert warnings to annotations
```

* Say what warnings are being converted to (at first I assumed to errors)
* Fix comma splice